test/org/opensolaris/opengrok/analysis/perl/PerlXrefTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2012, 2016, Oracle and/or its affiliates. All rights reserved.
```

Comment:
you create new files, so this should be just 2017  (not 2012, 2016)
same for all other new files, license basically traces since when does the copyright start

Reply:
Actually, I don't think the year needs bumping. Rather, @idodeclare should add his own copyright.

Reply:
This was not my all my original work. I file-copied PhpXrefTest.java — hence the inherited copyright — and then reworked it for PerlXref — hence my own contributor notice.